lib/games_statistics.rb

```diff
@@ -0,0 +1,112 @@
+require "CSV"
+require_relative "./games"
+require_relative "./teams"
+require_relative "./game_teams"
+
+class GamesStatistics
+  attr_reader :games, :game_teams, :teams
+  
+  def self.from_csv(locations)
```

Comment:
Stat tracker should be the only class that is calling self.from_csv
This is what the top of my class looks like for team_statistics.
```ruby
require "./module/calculatable"
class TeamStatistics
  include Calculatable

  def initialize(games, teams, game_teams)
    @games = games
    @teams = teams
    @game_teams = game_teams
  end
```

And this is what the test set up method needed to look like for it to work as we set it out on the stat_tracker:
```ruby
class TeamStatisticsTest < Minitest::Test
  def setup
    game_path = './data/games.csv'
    team_path = './data/teams.csv'
    game_teams_path = './data/game_teams.csv'

    locations = {
      games: game_path,
      teams: team_path,
      game_teams: game_teams_path
    }
    @stat_tracker = StatTracker.from_csv(locations)
    @games = @stat_tracker.games
    @teams = @stat_tracker.teams
    @game_teams = @stat_tracker.game_teams

    @team_statistics = TeamStatistics.new(@games, @teams, @game_teams)
  end
```